### PR TITLE
[Console-IoT] Add id to connection module

### PR DESCRIPTION
**Pull request by suyash-naithani**
### Type of change

- Refactoring

### Description

 Add meaningful ID to connections component

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx [hunk 1]
```diff
@@ -70,11 +70,11 @@ const ConnectionsToolbar: React.FunctionComponent<IConnectionsToolbarProps &
   ];
   const dropdownItems = [
     <DropdownItem
-      id="cl-filter-dropdown-item-closeAll"
       key="close-all"
       value="closeAll"
       component="button"
       isDisabled={selectedConnections.length <= 0}
+      id="connect-toolbar-close-selected-dropdown"
```
Comment (ajaypratap003) [inline console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx:77]: replace connect with connections for all file. This is  dropdwonitem not dropdown, so use dropdownitem for dropdwonitem. It shpould be 
id="connections-toolbar-close-selected-dropdownitem"
Please follow same for other ids.

File: console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx [hunk 2]
```diff
@@ -121,8 +121,8 @@ const ConnectionsToolbar: React.FunctionComponent<IConnectionsToolbarProps &
       </ToolbarItem>
       <ToolbarItem data-codemods="true">
         <DropdownWithKebabToggle
-          id="cl-select-kebab-overflow-dropdown-"
-          toggleId="cl-filter-overflow-kebab"
+          id="connect-toolbar-filter-dropdown-"
```
Comment (ajaypratap003) [inline console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx:124]: it should be id="connections-toolbar-filter-dropdown"

File: console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx [hunk 3]
```diff
@@ -133,7 +133,7 @@ const ConnectionsToolbar: React.FunctionComponent<IConnectionsToolbarProps &
 
   return (
     <Toolbar
-      id="data-toolbar-with-filter"
+      id="connect-toolbar-filter"
```
Comment (ajaypratap003) [inline console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx:136]: id="connections-toolbar-filter"

File: console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx (hunk 1 above)
Comment (suyash-naithani) [reply console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx]: Implemented change.

File: console/console-init/ui/src/modules/connection/components/ConnectionsToggleGroup/ConnectionsToggleGroup.tsx [hunk 4]
```diff
@@ -91,6 +91,8 @@ const ConnectionsToggleGroup: React.FunctionComponent<IConnectionsToggleGroupPro
           {filterSelected && filterSelected.toLowerCase() === "hostname" && (
             <InputGroup>
               <TypeAheadSelect
+                id="connections-toggle-grp-select-hostname-typeahead"
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/connection/components/ConnectionsToggleGroup/ConnectionsToggleGroup.tsx:94]: We should avoid using terms like typeahead, as it's too patternfly specific. We should use generic terms like 'dropdown' or 'auto-populate' which is understandable to all.

File: console/console-init/ui/src/modules/connection/components/ConnectionsToggleGroup/ConnectionsToggleGroup.tsx [hunk 5]
```diff
@@ -133,7 +137,7 @@ const ConnectionsToggleGroup: React.FunctionComponent<IConnectionsToggleGroupPro
                 setInput={setHostContainerInput}
               />
               <Button
-                id="cl-filter-search"
+                id="connections-togglegrp-search-addr-button"
```
Comment (anukritijha) [inline console/console-init/ui/src/modules/connection/components/ConnectionsToggleGroup/ConnectionsToggleGroup.tsx:140]: I don't think this button here searches for address. Can you clarify what exactly this button searches?

**Pull request merged**